src/components/UserProfile/UserIntegrations/IntegrationTableItem.js

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import styled from 'react-emotion';
+
+import CheckCircleIcon from 'icons/CheckCircleIcon';
+
+const Row = styled('tr')`
+  td {
+    padding: 16px;
+    font-weight: normal;
+    background: white;
+    height: 100%;
+
+    div.wrapper {
+      height: 100%;
+
+      display: flex;
+      justify-content: center;
+    }
+  }
+
+  td.checkmark {
+    padding: 0px;
+    width: 44px;
+
+    div {
+      display: flex;
+      align-items: center;
+    }
+
+    svg {
+      flex: 1;
+    }
+  }
+
+  td.logo {
+    padding: 0px;
+    width: 136px;
+
+    img {
+      max-height: 48px;
+      max-width: 136px;
+      object-fit: contain;
+    }
+  }
+
+  td.actions {
+    width: 363px;
+
+    background: ${({ theme }) => theme.backgroundGrey};
+  }
+`;
+
+export default ({ connected = false, logo, description, actions, ...props }) => {
+  return (
+    <Row {...props}>
```

Comment:
Not sure what else other than `className` needs to be passed down here, if nothing, I'd suggest going without `...props`. 
Passing unnecessary props down can cause weird errors so just wanna avoid `...props` where possible :)

Reply:
👍